index.html

```diff
@@ -3,10 +3,671 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="styles.css">
+    <link type="text/css" rel="stylesheet" href="styles.css"> 
     <title>TNW</title>
 </head>
 <body>
+
+        <header>
+            <div class="logo-breakpoint">
+                <img  src="./graphics/tnw-w.png" alt="">
+            </div>
+
+            <div class="hidden-nav flex">
+            <ul class="left-hidden-nav">
+                <li><a href="#">News</a></li>
+                <li><a href="#">Events</a></li>
+                <li><a href="#">Business</a></li>
+                <li><a href="#">AMAs</a></li>
+                <li><a href="#">Spaces</a></li>
+                <li><a href="#">Terms & Conditions</a></li>
+            </ul>
+            <ul class="right-hidden-nav">
+                <li><a href="#">About</a></li>
+                <li><a href="#">Advertise</a></li>
+                <li><a href="#">Jobs</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
+        </div>
+        </header>
+
+        <nav class="sticky-nav flex">
+            <div class="logo flex">
+                <img  src="./graphics/tnw-w.png" alt="">
+            </div>
```

Comment:
- [ ] In desktop, the logo doesn't appear, please hide it with media queries
![image](https://user-images.githubusercontent.com/53256139/88866435-4fc5fe80-d1d0-11ea-8b58-85c05607ef72.png)